script.js

```diff
@@ -1,133 +1,81 @@
-// 3. Make your function’s humanChoice parameter case-insensitive so that players can input “rock”, “ROCK”, “RocK”, or other variations.
-// 4. Write the code for your playRound function to console.log a string value representing the round winner, such as: “You lose! Paper beats Rock”.
-// 5. Increment the humanScore or computerScore variable based on the round winner.
-
-// Partially done compare function
-// function playRound(humanChoice, computerChoice) {
-//   humanChoice.toLowerCase();
-//   computerChoice.toLowerCase();
-//   if (humanChoice == 'rock' && computerChoice == 'paper') {
-//     console.log("You lose! Paper beats Rock");
-//   } else if (humanChoice == 'rock' && computerChoice == 'scissor') {
-//     console.log("You win! Rock beats Scissor");
-//   } else if (humanChoice == 'paper' && 'scissor') {
-//     console.log("You lose! Scissor beats Paper")
-//   } else if (humanChoice == 'paper' && computerChoice == 'rock') {
-//     console.log("You win! Paper beats rock")
-//   } else console.log("Incomplete");
-// }
+// debug log
+// console.log(getHumanChoice());
 
-// Function with complete win, lose and tie logic
-function playRound(humanChoice, computerChoice) {
-  humanChoice.toLowerCase();
-  computerChoice.toLowerCase();
-  if (humanChoice == 'rock' && computerChoice == 'paper' || humanChoice == 'paper' && computerChoice == 'scissor') {
-    console.log(`You lose! ${computerChoice} beats ${humanChoice}`);
-    computerScore = +1;
-    console.log(`Computer score is ${computerScore}`);
-  } else if (humanChoice == 'rock' && computerChoice == 'scissor' || humanChoice == 'paper' && computerChoice == 'rock' || humanChoice == 'scissor' && computerChoice == 'paper') {
-    console.log(`You win! ${humanChoice} beats ${computerChoice}`);
-    humanScore = +1;
-    console.log(`Human score is ${humanScore}`);
-  } else if (humanChoice == computerChoice) {
-    console.log("Tie! None gets a point");
+// Add logic to play an entire game
+// Game will be 5 rounds, write function playGame that calls playRound, keeps scores and declares winner.
+// Create a new function named playGame.
+// Move your playRound function and score variables so that they’re declared inside of the new playGame function
+// Play 5 rounds by calling playRound 5 times.
+
+//     Hint: When you assign a function call to a variable, the return value of that function is assigned to the variable. Accessing the variable afterward will only provide the assigned value; it doesn’t recall the function. You need to recall the choice functions to get new choices for each round.
+
+function playGame() {
+  // Declare score values
+  let humanScore = 0;
+  let computerScore = 0;
+  // Loop function, run it 5 times
+  for (let step = 0; step <5; step++) {
+  // Function that has the main game logic
+  function playRound(humanChoice, computerChoice) {
```

Comment:
This is weird: you're writing 
```js
for (...) {
  function playRound(...){
    ...
  }
  playRound();
}
```

but you should write

```js
function playRound(...){
  ...
}

for (...) {
  playRound();
}
```

Reply:
Yeah, in your code you are re-creating the function `playRound` on each round, instead of creating the function once and using it 5 times :)

Reply:
But if it doesn't work then it's not a big deal, you should move on to another exercise :)